src/src/test/java/org/niis/xroad/restadapterservice/ConsumerGatewayIT.java

```diff
@@ -53,6 +66,8 @@
 @Slf4j
 public class ConsumerGatewayIT {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(ConsumerGatewayIT.class);
```

Comment:
Should use lombok \@slf4j generated logger instead of defining extra logger

Reply:
fixed